src/components/Header/Pages.js

```diff
@@ -1,6 +1,6 @@
 import strings from 'lang';
```

Comment:
if this is just going to export navbar pages, it should probably be renamed.  Or just inline the pages in the component that uses it (if there's only one).